qa/pull-tester/rpc-tests.py

```diff
@@ -76,11 +67,24 @@
 if "BITCOINCLI" not in os.environ:
     os.environ["BITCOINCLI"] = buildDir + '/src/bitcoin-cli' + EXEEXT
 
-#Disable Windows tests by default
 if EXEEXT == ".exe" and "-win" not in opts:
-    print "Win tests currently disabled.  Use -win option to enable"
+    # https://github.com/bitcoin/bitcoin/commit/d52802551752140cf41f0d9a225a43e84404d3e9
```

Comment:
Makes sense to refer to the discussion of the pull this was part of: https://github.com/bitcoin/bitcoin/pull/5677#issuecomment-136646964